packages/zoe/test/unitTests/contracts/test-throwInOfferHandler.js

```diff
@@ -0,0 +1,45 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import '@agoric/install-ses';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import test from 'ava';
+
+import bundleSource from '@agoric/bundle-source';
+
+import { E } from '@agoric/eventual-send';
+import { makeZoe } from '../../../src/zoeService/zoe';
+import fakeVatAdmin from '../../../src/contractFacet/fakeVatAdmin';
+
+const contractRoot = `${__dirname}/throwInOfferHandler`;
+
+test('throw in offerHandler', async t => {
```

Comment:
It doesn't seem like this test adds anything to what brokenContracts tests. That test (throwInOfferHook) verifies that the parties get their deposits back and that zoe isn't hosed. 

Are there other constraints we should verify we aren't violating?

Reply:
It tests that an error in the `offerHandler` within `depositToSeat` is handled correctly, which brokenContracts does not do, which is how we got in this situation to start with.

It also tests that an offerHandler throwing generally is handled correctly. This is tested in brokenContracts, but in swingset tests that are very slow. I think it's worth having both a fast and a slow Swingset version.

Reply:
Okay. I'm good with that.  My prior approval stands.